src/util/language_file.cpp

```diff
@@ -11,6 +11,10 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "language.h"
 #include "language_file.h"
 
+// Defined here so that unique_ptr works properly
```

Comment:
The pointed-to type must be complete at the point at which the compiler attempts to create a destructor for the `unique_ptr`. If the destructor is defined in the header, and only a forward declaration of the pointed-to type is available, the compiler will complain. By explicitly placing the destructor in the source file, the complete definition of the pointed-to type is made visible, and the compiler will be happy. There's a better explanation in item 22 of 'Effective Modern C++'.